scout/server/blueprints/variants/utils.py

```diff
@@ -1,23 +1,20 @@
 # -*- coding: utf-8 -*-
+from scout.adapter import MongoAdapter
 
 
-def update_case_panels(store, case_obj):
+def update_case_panels(store: MongoAdapter, case_obj: dict):
```

Comment:
This gets called immediately before update_variant_case_panels. And loops thru all gene panels of the case anyway, so this makes so that the latest panels of a case with relative genes are all collected under case["latest panels"] and used to do comparisons with variants genes downstream.

Reply:
Yes, trade a bit of storage on the case level for speed when looping over many variantS.